src/main/java/nl/cherement/jak/controller/BoardController.java

```diff
@@ -21,7 +21,7 @@
 
     @GetMapping
     public List<BoardEntity> findAll(Principal principal) {
-        return service.findByUserName(principal.getName());
+        return service.findBoardByUserName(principal.getName());
```

Comment:
Is dit nodig aangezien het in de boardController zit?

Reply:
Ja dit is nodig. Omdat een user op een board staat.
En dus de betrefende boards van de user met de user via de boardservice opgevraagt word.

Reply:
Nee ik ben het eigenlijk met Jazzlyn eens had het hier niet nodig is. Ik was zelf hier begonnen met de security refactor proberen te doen. Maar toen ik een idee kreeg van hoeveel werk t was mee gestopt. Dit is een overblijfsel daarvan en ik zet het weer goed.